src/main/java/uk/ac/ebi/ena/webin/cli/manifest/ManifestReader.java

```diff
@@ -297,6 +321,87 @@ public void addListener(MessageListener listener) {
         return null;
     }
 
+    private void parseManifestNew( Path inputDir, List<String> lines ) {
+        state.state = PARSE;
+
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
+
+            JsonNode jsonNode = objectMapper.readTree(lines.stream().collect(Collectors.joining("\n")));
+
+            jsonNode.fields().forEachRemaining(field -> {
+                String fieldName = field.getKey();
+
+                if( fieldName.matches( FIELD_NAME_REGEX ) )

```

Comment:
- The FIELD_NAME_REGEX applies only for keyValue parser.
- The FIELD_NAME_REGEX pattern matches <whitespace><key name<other stuff>. It is used only to get the <key name part>.
- For JSON there is no business reason whatsoever to use the  FIELD_NAME_REGEX because we expect the field name from a certain 'place' in the JSON document.
- We should remove:
```
if( fieldName.matches( FIELD_NAME_REGEX ) ) return;
```